samples/ask-akka-agent/src/main/resources/flat-doc/deploy-service.md

```diff
@@ -158,7 +158,7 @@ Once pushed, you need to use the suggested image from the command’s output in
 
 ```yaml
 name: my-service
-spec:
+service:
```

Comment:
```suggestion
spec:
```

Reply:
reverting this, since it is generated content in a sample and will conflict with another PR